inst/manuscript/R/00-standalone-Figure-replication.R

```diff
@@ -143,7 +143,7 @@ pit_plots <- plot_pit(stored$pit) +
 # create interval and quantile coverage plots ----------------------------------
 # create coverage plots by transforming to quantile format first
 quantiles <- c(0.01, 0.025, seq(0.05, 0.95, 0.05), 0.975, 0.99)
-df_quantile <- sample_to_quantile(df,
+df_quantile <- as_forecast_quantile(df,
                                   quantiles = quantiles)
```

Comment:
wrong arg name here - we may want to add a depreciation check to watch for this and warn?

Reply:
I realised that both scripts you pointed to are just outdated and broken. Opened a new issue: https://github.com/epiforecasts/scoringutils/issues/881. 
I suggest addressing them outside of this PR. Only question is before or after CRAN release. I guess before? 🥲